src/ints/int10_modes.cpp

```diff
@@ -2292,7 +2292,9 @@ static cga_colors_t handle_cga_colors_prefs_tandy(const std::string& cga_colors_
 	if (tokens.size() > 1) {
 		auto brown_level_pref = tokens[1];
 
-		if (float p; sscanf(brown_level_pref.c_str(), "%f", &p)) {
+		float p = {};
```

Comment:
Could you use `parse_float()` in this file?

https://github.com/dosbox-staging/dosbox-staging/blob/d6b11b9752304ee46c9e059e58bec42db6b9e911/src/utils/string_utils.cpp#L290